src/commands/azure/deleteRegistry.ts

```diff
@@ -7,17 +7,17 @@ import { Registry } from "azure-arm-containerregistry/lib/models";
 import { SubscriptionModels } from "azure-arm-resource";
 import * as vscode from "vscode";
 import { IActionContext } from "vscode-azureextensionui";
-import { AzureRegistryNode } from '../../explorer/models/azureRegistryNodes';
+import { AzureRegistryNode } from '../../../explorer/models/azureRegistryNodes';
 import { ext } from "../../extensionVariables";
 import * as acrTools from '../../utils/Azure/acrTools';
 import { AzureUtilityManager } from '../../utils/azureUtilityManager';
 import { nonNullProp } from "../../utils/nonNull";
-import { confirmUserIntent, quickPickACRRegistry } from '../utils/quick-pick-azure';
+import { confirmUserIntent, quickPickACRRegistry } from '../../utils/quick-pick-azure';
 
 /** Delete a registry and all it's associated nested items
  * @param node : the AzureRegistryNode the user right clicked on to delete
  */
-export async function deleteAzureRegistry(_context: IActionContext, node?: AzureRegistryNode): Promise<void> {
+export async function deleteRegistry(_context: IActionContext, node?: AzureRegistryNode): Promise<void> {
     let registry: Registry;
```

Comment:
Should this be Azure registry?  Were you trying to just make everything consistent?